### ISSUE TITLE
Marshmallow

### ISSUE THREAD

**Issue by BraisGabin** [phajduk/RxFileObserver#1]
Hi, you are probably familiar with this issue: https://code.google.com/p/android/issues/detail?id=189231. I tried your library and it works great, but in Marshmallow phones it do nothing.

Do you know any workaround? My workaround is pooling a directory but it have some issues: https://github.com/BraisGabin/pokescreenshot/blob/master/app/src/main/java/com/braisgabin/fileobservable/FileObservable.java

One of the issues is that I detect the creation of the file, but at that point the screenshot process didn't end to write the file. I have an open question about this issue too: http://stackoverflow.com/q/39878517/842697

Well... I think that this issue is more a question than a real issue... But I think that any user of RxFileObserver must know about this problem.

A big warning (or even a crash) if you run this library on a Marshmallow Device would be great. And if the message points out to any web that describes the problem and gives some solutions would be fantastic.

**Comment by xstar97** [comment]
@BraisGabin 
dude i dont think the dev will even help since its technically out of his control & he hasnt done anything to this library for over entire year...so...google is your friend to find another solution for SDK23.

**Comment by BraisGabin** [comment]
Yes, I think so too. My infinite-one-second-loop is going to run for now. Anyway I hope that this issue and/or the SO question will help someone to waste less time than me in this issue.

**Comment by magillus** [comment]
There is latest "small" work around posted on:
https://code.google.com/p/android/issues/detail?id=189231

> In my tests FileObserver seems to observe only folders, not files. It further seems to only catch events of files in the observed folder, not on subfolders.
> 
> So I think the only thing you can never observe is a folder being created/deleted. The rest you can observe with a workaround by observing the parent folder.
